docs/language-server/protocol-project-manager.md

```diff
@@ -750,6 +751,50 @@ interface ProjectStatusResponse {
 }
 ```
 
+### `project/duplicate`
+
+This message requests to make a copy of the project.
+
+- **Type:** Request
+- **Direction:** Client -> Server
+- **Connection:** Protocol
+- **Visibility:** Public
+
+#### Parameters
+
+```typescript
+interface ProjectDuplicateRequest {
+  /**
+   * The project to duplicate.
+   */
+  projectId: UUID;
```

Comment:
Shouldn't the message support (optional) suggested name for the newly created project?

Reply:
In the cloud, the project with the ` (copy)` suffix is created, and then the user can rename it.